.travis.yml

```diff
@@ -1,24 +0,0 @@
-language: python
-sudo: required
-python:
-  - "3.5"
-  - "3.6"
-  - "3.7"
-  - "3.8"
-env: TEST_SUITE=test-main
-matrix:
-  include:
-  - python: "3.6"
-    env: TEST_SUITE=test-static-analysis
-    addons:
-      postgresql: "9.3"
```

Comment:
Is this relevant to the new version?

Reply:
I removed Postgres because it didn't need any Postgres instance while setup and probably uses, mock for testing those that need Postgres. (In short, all the tests run without Postgres).